src/Extension/SEODataExtension.php

```diff
@@ -164,7 +164,7 @@ public function SEOData()
 		];
 	}
 
-	public function MetaTagCollection($raw = false)
+	public function MetaTagCollection($raw = false, $includeMetaTitle)
```

Comment:
Always define the arguments without default values before the ones with defaults.